unitTests/ui/DynamicFacetManagerTest.ts

```diff
@@ -18,25 +19,22 @@ export function DynamicFacetManagerTest() {
     beforeEach(() => {
       options = {};
       initializeFacets();
-      initializeComponent();
-      test.cmp.ensureDom();
+      initializeManager();
     });
 
     function initializeFacets() {
       facets = [
-        DynamicFacetTestUtils.createAdvancedFakeFacet({ id: 'test1', field: '@test1', numberOfValues: 10 }).cmp,
```

Comment:
Related to the comment above, I'm curious why you made this change?

Reply:
I added the method initializeManyFacets and didn't want to specify a field for each, but I guest I could just use the index as well